subprojects/packagefiles/xtensor-python/meson.build

```diff
@@ -0,0 +1,45 @@
+project('xtensor-python', 'cpp',
+        version:'0.24.1',
+        license:'BSD-3-Clause',
+        default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+pybind_dep = dependency('pybind11',
+    version: ['>=2.4.3', '<3.0.0'])
+
+xtensor_dep = dependency('xtensor',
+    version: ['>=0.21.2', '<1.0.0'])
+
+python = import('python').find_installation('python3')
+
+npy_incstr = run_command(
+    [python, '-c', 'import numpy; print(numpy.get_include())'],
+    check: false).stdout().strip()
+
+if (npy_incstr == '')
+    warning('Unable to find numpy (Looking for it using ' + python.path() +
+        '), disabling python bindings.')
+endif
+
+extra_cflags = ['-I' + npy_incstr]
+
+xtensor_python_dep = declare_dependency(
+    include_directories: [include_directories('include', is_system: true),
+        include_directories(npy_incstr, is_system: true)],
```

Comment:
nitpicking: indentation seems off here.

Reply:
I don't think it is...? Just happens to look that way because of the repeated `include_directories`

Reply:
Ah sorry, it's an item in the list not the next kwarg indeed. I would align both `include_directories()` but I don't think we have any strong coding style, so I guess we can leave it.